src/common/format/common.fbs

```diff
@@ -20,6 +20,11 @@ table ResourcePair {
   value: double;
 }
 
+enum TaskLanguage:int {
```

Comment:
I named this as `TaskLanguage`, because `Language` was already used by https://github.com/ray-project/ray/blob/9559873d135de78734a835d9725f2c0dabe4ace7/src/ray/gcs/format/gcs.fbs#L1. 

But that enum seems not being used anywhere. Is that still needed? @pcmoritz

Reply:
We should only have one of these enums, feel free to remove one of them!

Reply:
@pcmoritz thanks for replying. I tried to keep only one enum, but it seemed complex. Thus, I ended up still keeping 2 enums. See these 2 comments for details https://github.com/ray-project/ray/pull/2639#issuecomment-412502977 https://github.com/ray-project/ray/pull/2639#issuecomment-412803008.

Do you have any better ideas?

Reply:
Agreed. Once we get rid of legacy Ray, we should move the task specification to  `src/ray` and let `TaskTableData` reference it directly https://github.com/ray-project/ray/blob/dd924a388b774af66a06d72e5ada3f52bc2912ad/src/ray/gcs/format/gcs.fbs#L88.

Also, [local_scheduler.fbs](https://github.com/ray-project/ray/blob/master/src/local_scheduler/format/local_scheduler.fbs) needs to be removed as well. It's duplicate with [node_manager.fbs](https://github.com/ray-project/ray/blob/master/src/ray/raylet/format/node_manager.fbs)